ekncontent/ekncontent/eknc-object-model.c

```diff
@@ -24,6 +25,8 @@ EkncContentObjectModel *
 eknc_object_model_from_json_node (JsonNode *node,
                                   GError **error)
 {
+  g_autoptr(EosProfileProbe) probe = EOS_PROFILE_PROBE ("/ekncontent/object_model");
```

Comment:
nit: maybe there is a better name for probe? To me, `object_model` didn't suggested that it was this helper.. if the idea is to probe the common point for all model objects then maybe something like `/ekncontent/object`, or we want to make it very explicit then just `/ekncontent/object/from_json_node` or something.

Reply:
I went with `object/from_json_node`, as it makes it clear where the probe is applied (outside of the location).